PopupMenuDemo/MainActivity.cs

```diff
@@ -0,0 +1,34 @@
+namespace PopupMenuDemo;
+
+[Activity(Label = "PopupMenuDemo", MainLauncher = true)]
+public class Activity1 : Activity
+{
+    protected override void OnCreate(Bundle? bundle)
+    {
+        base.OnCreate(bundle);
+
+        SetContentView(Resource.Layout.Main);
+
+        Button showPopupMenu = FindViewById<Button>(Resource.Id.popupButton)!;
+            
+        showPopupMenu!.Click += (s, arg) => {
+
+        PopupMenu menu = new PopupMenu(this, showPopupMenu);
```

Comment:
You might use `var` here instead of `Button` and `PopupMenu`.

The only place I *wouldn't* use `var`, would be something like:

```csharp
var something = AMethodThatDoesntShowReturnType();
```
In this case, you can't read it on GitHub and know what `var` is ^^